src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -50,8 +49,13 @@ private void ShowGoldBarForCodeCleanupConfigurationIfNeeded(Document document)
             var optionPageService = document.GetLanguageService<IOptionPageService>();
             var infoBarService = document.Project.Solution.Workspace.Services.GetRequiredService<IInfoBarService>();
 
+            // wording for the Code Cleanup infobar will be different depending on if the feature is enabled 
+            var infoBarMessage = docOptions.GetOption(CodeCleanupOptions.PerformAdditionalCodeCleanupDuringFormatting)
+                    ? EditorFeaturesResources.Format_document_performed_additional_cleanup
```

Comment:
Do we care the scenario that the cleanup is done but no code style change?  we say "performed additional cleanup" but actually no additional cleanup here.